Source/DiscoveryV1/Discovery.swift

```diff
@@ -3215,4 +3241,315 @@ public class Discovery {
         }
     }
 
+    /**
+     List credentials.
```

Comment:
I noticed that a lot of methods have this line in the API documentation that is redundant with the more detailed description below. That's something we should probably fix in the generator.